packages/harmony/src/components/button/BaseButton.module.css

```diff
@@ -0,0 +1,43 @@
+/* ===Base Styles=== */
+.button {
+  align-items: center;
+  box-sizing: border-box;
+  cursor: pointer;
+  display: inline-flex;
+  flex-shrink: 0;
+  justify-content: center;
+  overflow: hidden;
+  position: relative;
+  text-align: center;
+  user-select: none;
+  white-space: nowrap;
+  transition: all var(--hover);
```

Comment:
i dont think the transition css vars are in harmony so might need to add those for this and the --press one below